tests/unit/specs/components/forms/Scopes.spec.js

```diff
@@ -66,21 +48,9 @@ describe('Scopes', () => {
         isLoading: false,
         scopesList: ['foo', 'bar'],
       });
-      wrapper.vm.onChange({ foo: true, bar: true });
       wrapper.find('form').trigger('submit');
 
       expect(wrapper.emitted().submit).toEqual([[['foo', 'bar']]]);
```

Comment:
be sure that this event wasn't called before you trigger submit
`expect(wrapper.emitted().submit).toBeUndefined()`